api/src/handlers/authorizer.py

```
@@ -125,24 +108,24 @@ def lambda_handler(event: Dict[str, Any], context) -> Dict[str, Any]:
                         'authMethod': 'hmac-sha256'
                     }
                 )
             else:
-                print("AUTHORIZER: Authentication failed")
+                logger.warning("Request authentication failed")
                 return generate_policy('unauthenticated', 'Deny', event['methodArn'])
-                
+
         except Exception as auth_error:
-            print(f"AUTHORIZER: Auth system error: {str(auth_error)}")
+            logger.error(f"Auth system error: {str(auth_error)}")
```

Ruff (0.15.21)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)